testing/constraints-3.6.txt

```diff
@@ -11,6 +11,5 @@ google-auth==1.25.0
 requests==2.18.0
 setuptools==40.3.0
 packaging==14.3
-grpcio==1.29.0
-grpcio-gcp==0.2.2
+grpcio==1.33.2
```

Comment:
Here the repetition is OK, because we are intentionally testing the lowest bound of the range.

Reply:
Cool 👍

Reply:
Let me know if I should put it back or not.

Reply:
@lidizheng We do have explicit tests for including the `grpcio-gcp` package -- I think it needs to remain pinned here.  Bumping the pin for `grpcio` should be fine.